conf/db.sql

```diff
@@ -98,25 +98,24 @@ DROP TABLE IF EXISTS `Emails`;
 /*!40101 SET character_set_client = utf8 */;
 CREATE TABLE `Emails` (
   `EmailID` bigint(20) NOT NULL AUTO_INCREMENT,
-  `UserID` bigint(20) NOT NULL DEFAULT '0',
+  `UserID` bigint(20) NOT NULL DEFAULT 0,
   `Prefix` varchar(20) DEFAULT NULL,
   `Word` varchar(20) NOT NULL DEFAULT '',
-  `InitialCount` tinyint(4) NOT NULL DEFAULT '0',
-  `Count` tinyint(4) NOT NULL DEFAULT '0',
+  `InitialCount` tinyint(4) NOT NULL DEFAULT 0,
+  `Count` tinyint(4) NOT NULL DEFAULT 0,
   `TimeAdded` bigint(20) DEFAULT NULL,
   `Sender` varchar(254) DEFAULT NULL,
-  `Address` text,
-  `DeliveryAddress` varchar(255) DEFAULT '',
-  `Hidden` tinyint(4) NOT NULL DEFAULT '0',
+  `Address` text DEFAULT NULL,
+  `Hidden` tinyint(4) NOT NULL DEFAULT 0,
```

Comment:
`DeliveryAddress` has disappeared and `Address` now defaults to `NULL`. This change might actually belong in #39.

Reply:
This change moved to #39.